vocode/streaming/agent/default_factory.py

```diff
@@ -23,4 +25,6 @@ def create_agent(self, agent_config: AgentConfig) -> BaseAgent:
             return RESTfulUserImplementedAgent(agent_config=agent_config)
         elif isinstance(agent_config, AnthropicAgentConfig):
             return AnthropicAgent(agent_config=agent_config)
+        elif isinstance(agent_config, LangchainAgentConfig):
```

Comment:
let's keep this out of the default factory, because it will require the user to install the langchain / langchain-extras deps